core/src/test/java/org/apache/druid/timeline/partition/DimensionRangeShardSpecTest.java

```diff
@@ -42,264 +39,6 @@
 
   private final List<String> dimensions = new ArrayList<>();
 
-  @Test
-  public void testIsInChunk()
```

Comment:
Please don't remove these tests altogether as there is nothing else testing this behaviour.
You can convert these tests to test `getLookup` instead.

Reply:
You could create multiple shard specs, representing adjacent (or non-adjacent too if you are testing for it) partitions. Then create a `ShardSpecLookup` using the `getLookup` method. Then convert the assertions accordingly.

Reply:
done